### PR TITLE
[Node client] Auto-generated bindings

### PR DESCRIPTION
**Pull request by batiati**
This PR includes code for auto-generating typescript bindings for the node client.

Also, other minor improvements:

- Fixes `/src/node/scripts/benchmark.sh` script;
- Standardizes the way clients' bindings are validated;
- Remove an obsolete TODO regarding a `tracer_backend` hack on `node.zig`.

## Pre-merge checklist

Performance:

* [X] I am very sure this PR could not affect performance.

### CODE REVIEW

**Comment by eatonphil** [comment]
Very nice!

**Comment by eatonphil** [comment]
The lint build failure is my fault, trying to fix. https://github.com/tigerbeetledb/docs/pull/13

**Comment by eatonphil** [comment]
I was looking in the wrong place. Hopefully this does it: https://github.com/tigerbeetledb/tigerbeetle/pull/601.

File: src/clients/node/src/index.ts [hunk 1]
```diff
@@ -1,3 +1,12 @@
+export * from './bindings'
```
Comment (batiati) [inline src/clients/node/src/index.ts:1]: I need a TypeScript expert opinion here:
Is it the idiomatic way to import and re-export declarations?

Comment (sentientwaffle) [reply src/clients/node/src/index.ts:1]: I am not a TypeScript expert, but it does look right to me.

**Review by sentientwaffle — APPROVED** [review]

**Pull request merged**